src/main/java/org/broadinstitute/consent/http/db/mapper/DarCollectionSummaryReducer.java

```diff
@@ -44,6 +49,28 @@ public void accumulate(Map<Integer, DarCollectionSummary> map, RowView rowView)
           summary.addReferenceId(darReferenceId);
         }
 
+        if (hasColumn(rowView, "dar_all_reference_id", String.class)) {
+          aggReferenceId = rowView.getColumn("dar_all_reference_id", String.class);
+          if (Objects.nonNull(aggReferenceId)) {
+            summary.addReferenceId(aggReferenceId);
+            if (hasColumn(rowView, "dar_all_parent_id", String.class)) {
+              aggParentId = rowView.getColumn("dar_all_parent_id", String.class);
```

Comment:
Pulling in as String to be consistent with our other implementation.  Knowing we're going back the the UUID referenceId made me do this now, but if we want this released before we're going to do that work, I would change to Integer so the contents work as expected on the front end.